packages/rocketchat-livechat/server/methods/removeRoom.js

```diff
@@ -0,0 +1,31 @@
+Meteor.methods({
+	'livechat:removeRoom'(rid) {
+		if (!Meteor.userId() || !RocketChat.authz.hasPermission(Meteor.userId(), 'view-livechat-manager')) {
```

Comment:
instead of checking permission for `view-livechat-manager` can you please change to something like `remove-closed-livechat-rooms` and add this permission to the `livechat-agent` role (and to admin as well)? a migration might be needed.

this way we can admins can for example remove this ability from livechat managers, so only admins could do it.

Reply:
oh sry, you're absolutely right.. what I meant to say was `livechar-manager` role (and not `livechat-agent`).. I'll edit my comment to reflect that. thx

Reply:
@sampaiodiego, just to be sure, I guess changing the `view-livechat-manager` it will impact here as well:

https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-livechat/client/views/app/livechatCurrentChats.html#L2